cg/models/demultiplex/flow_cell.py

```diff
@@ -126,6 +126,28 @@ def sequencer_type(
         """Return the sequencer type."""
         return sequencer_types[self.machine_name]
 
+    def set_bcl_converter(self, bcl_converter: str) -> str:
+        """
+        Return the bcl converter to use.
+        Tries to fetch the bcl converter from the sequencer type if not provided.
+        """
+        if bcl_converter:
+            return bcl_converter
+        return self.get_bcl_converter_by_sequencer()
```

Comment:
```suggestion
        return bcl_converter or self.get_bcl_converter_by_sequencer()
```
I think you can use a ternary operator here